.bazelversion

```diff
@@ -1 +1 @@
-3.3.1
+4.0.0
```

Comment:
I don't think this belongs here?
If we want to bump our bazel version it should be in a separate PR

Reply:
I commented that on the PR

https://github.com/bazelbuild/rules_python/pull/524#issuecomment-913681103

Reply:
I see. Would be easier for me to see them in the right order :)
Could you send that PR first and let's resolve that issue?

Reply:
Opened https://github.com/bazelbuild/rules_python/pull/533